pkg/services/sqlstore/sqlstore.go

```diff
@@ -37,6 +37,8 @@ var (
 const ContextSessionName = "db-session"
 
 func init() {
+	xorm.DefaultPostgresSchema = ""
```

Comment:
I think this deserves a comment explaining why.

Reply:
Git commit?

Reply:
```suggestion
	// This change will make xorm use an empty default schema for postgres and
	// by that mimic the functionality of how it was functioning before
	// xorm's changes above.
	xorm.DefaultPostgresSchema = ""
```